### ISSUE TITLE
Can't download the video

### ISSUE THREAD

**Issue by bbdev-tahir** [mclmza/linkedin-learning-downloader#80]
i am getting this error on few vidoes "Can't download the video [video name], probably is only for premium users"

**Comment by emobile-dev** [comment]
This could be because linkedin removes the authentication token. You need to restart the process again.
make sure your account has premium access.

**Comment by bbdev-tahir** [comment]
Yes, i do have premium access and i tried it again. Most of videos are working fine but i am still getting same error on same videos.

**Comment by mclmza** [comment]
Can you post your config file and specify which courses give you problems?

**Comment by derblub** [comment]
I've encountered the same issue.  
One of those problematic courses has this slug: `photoshop-cc-2018-one-on-one-advanced`.  
The first chapter downloads without issues, but on the second chapter, only a few videos are working.  

When checking the returned json-data, you'll see that `video_data.json()['elements'][0]['selectedVideo']['url']` is missing entirely from those videos, while they still load and play on the web-ui.  

I'm still trying to figure out, where to get the correct video-urls..
